### ISSUE TITLE
Combine with pydocstring?

### ISSUE THREAD

**Issue by peterjc** [terrencepreilly/darglint#41]
It isn't as prominent in the the ``pydocstrings`` documentation as it should be, but they added basic numpydoc checking in v2.0.0 (April 2017, under setting ``convention=numpy``) and Google style in v4.0.0 (July 2019, under setting ``convention=google``):

http://www.pydocstyle.org/en/latest/release_notes.html

Is it realistic to port any of darglint's code into pydocstrings instead of maintaining it as a separate library?

Cross reference #28.

**Comment by terrencepreilly** [comment]
That's interesting that they're starting to support checking for arguments in the docstring.  When I set out with this project, I tried not to have any overlap with `pydocstyle`: looks like that's inevitable now.

I'd be surprised if they would be okay with porting darglint into pydocstyle.  It would probably introduce a significant performance regression they would not like.  It would probably be more realistic to request that darglint be added to the PyCQA project.

I probably won't do it, since that would be a lot of work, and I want to focus on developing features and increasing the quality of this project.  That said, if anyone wants to take anything from darglint and put it into `pydocstyle`, I'm totally fine with that, and would help out when I could.

Since it's not my project, I have no say in any kind of porting.  Feel free to leave any comments about it here, though.
